release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -121,6 +121,44 @@ io:println(nameText); // "DanBrown"
 
 ##### Language Server
 
+##### Ballerina Shell
+
+Ballerina Shell now supports redefining module-level declarations and variable declarations. `/remove NAMES` command can be used to remove one or multiple declarations from snippet memory.
+
+```ballerina
```

Comment:
Should we use bash here too?

Reply:
Im okay with both bash and ballerina. I used Ballerina because most of the code in the snippet is ballerina code.